core/system/src/mpu/vmpu_freescale_k64_aips.c

```diff
@@ -117,6 +109,16 @@ int vmpu_aips_add(uint8_t box_id, void* start, uint32_t size, UvisorBoxAcl acl)
     return 1;
 }
 
+void vmpu_aips_common_add(void)
+{
+    int box, slot;
+    for (box = 1; box <= g_vmpu_box_count; box++) {
```

Comment:
Thank you @avri-schneider ! I'm now sending a new PR that also fixes the issue you reported.